### DISCUSSION TITLE
Deleting a transaction inside a liability does not update the Amount Due

### DISCUSSION THREAD

**Discussion by nikhil-rajesh** [firefly-iii/firefly-iii#6507] [Firefly III] [answered]
I deleted a transaction inside a liability account and it does not seem to update the Amount Due. Is there anyway I can refresh the Amount Due? One hack I found is to create a new temp transaction into the liability and delete it later.

**Comment by JC5** [comment, marked as answer]
That's a bug, nice find! You can also edit the liability and change nothing, that also works.
